src/views/RepoDetailIssueView/RepoDetailIssueView.tsx

```diff
@@ -59,6 +59,25 @@ const RepoDetailIssueView: React.FC<RepoDetailIssueViewProps> = ({
 
   if (!issue) return null;
 
+  const onClick = (params1: any, params2: any) => {
+    const ym = params1.data[0];
+    let type = params2.seriesName;
+    if (type === 'open') {
+      type = 'created';
+    } else if (type === 'comment') {
+      type = 'updated';
+    } else {
+      type = 'closed';
+    }
```

Comment:
`seriesIndex` is more robust, please see https://github.com/hypertrons/hypertrons-crx/pull/503#issue-1437321929.